src/probnum/problems/zoo/quad/_quadproblems_gaussian.py

```diff
@@ -0,0 +1,226 @@
+
+from probnum.problems import QuadratureProblem
+from probnum.quad.integration_measures import GaussianMeasure
+from probnum.typing import FloatLike
+
+__all__ = [
+    "uniform_to_gaussian_quadprob",
+    "sum_polynomials",
+]
+
+
+# Construct transformation of the integrand
+def uniform_to_gaussian_integrand(
+    fun: Callable[[np.ndarray], np.ndarray],
+    mean: Union[float, np.floating, np.ndarray] = 0.0,
+    std: Union[float, np.floating, np.ndarray] = 1.0,
+) -> Callable[[np.ndarray], np.ndarray]:
+    # mean and var should be either one-dimensional
+    if isinstance(mean, np.ndarray):
+        if len(mean.shape) != 1:
+            raise TypeError(
+                "The mean parameter should be a float or a d-dimensional array."
+            )
+
+    if isinstance(std, np.ndarray):
+        if len(std.shape) != 1:
+            raise TypeError(
+                "The std parameter should be a float or a d-dimensional array."
+            )
+
+    def new_func(x):
+        return fun(norm.cdf(x, loc=mean, scale=std))
+
+    return new_func
+
+
+def uniform_to_gaussian_quadprob(
+    quadprob: QuadratureProblem,
+    mean: Union[float, np.floating, np.ndarray] = 0.0,
+    std: Union[float, np.floating, np.ndarray] = 1.0,
+) -> QuadratureProblem:
+    r"""Creates a new QuadratureProblem for integration against a Gaussian on
+    :math:`\mathbb{R}^d` by using an existing QuadratureProblem whose integrand is
+    suitable for integration against the Lebesgue measure on :math:`[0,1]^d`.
+
+    The multivariate Gaussian is of the form :math:`\mathcal{N}(mean \cdot (1, \dotsc,
+    1)^\top, var^2 \cdot I_d)`.
+
+    Using the change of variable formula, we have that
+
+    .. math::  \int_{[0,1]^d} f(x) dx = \int_{\mathbb{R}^d} h(x) \phi(x) dx
+
+    where :math:`h(x)=f(\Phi((x-mean)/var))`, :math:`\phi(x)` is the Gaussian
+    probability density function and :math:`\Phi(x)` an elementwise application of the
+    Gaussian cummulative distribution function. See [1]_.
+
+    Parameters
+    ----------
+    quadprob
+        A QuadratureProblem instance which includes an integrand defined on [0,1]^d
+    mean
+        Mean of the Gaussian distribution. If `float`, mean is set to the same value
+        across all dimensions. Else, specifies the mean as a d-dimensional array.
+    std
+        Diagonal element for the covariance matrix of the Gaussian distribution. If
+        `float`, the covariance matrix has the same diagonal value for all dimensions.
+        Else, specifies the covariance matrix via a d-dimensional array.
+
+    Returns
+    -------
+    problem
+        A new Quadrature Problem instance with a transformed integrand taking inputs in
+        :math:`\mathbb{R}^d`.
+
+    Raises
+    ------
+    ValueError
+        If the original quadrature problem is over a domain other than [0, 1]^d or if it
+        does not have a scalar solution.
+
+    Example
+    -------
+        uniform_to_gaussian_quadprob(genz_continuous(1))
+
```

Comment:
This is not a valid docstring code example, since the syntax is invalid. See https://github.com/probabilistic-numerics/probnum/blob/e122f912a2483b8e3da28df083090afd385fad37/src/probnum/problems/zoo/linalg/_random_linear_system.py#L52-L64 for an example on how to create a docstring code example.